python/kserve/kserve/model_server.py

```diff
@@ -182,6 +212,13 @@ def __init__(
             enable_grpc: Whether to turn on grpc server. Default: ``True``
             enable_docs_url: Whether to turn on ``/docs`` Swagger UI. Default: ``False``.
             enable_latency_logging: Whether to log latency metric. Default: ``True``.
+            configure_logging: Whether to configure KServe and Uvicorn logging. Default: ``True``.
```

Comment:
You can remove these three arguments. This is no longer used. `access_log_format` is duplicated here.